includes/admin/class-wc-admin-notices.php

```diff
@@ -530,6 +546,32 @@ public static function simplify_commerce_notice() {
 	public static function theme_check_notice() {
 		wc_deprecated_function( 'WC_Admin_Notices::theme_check_notice', '3.3.0' );
 	}
+
+	/**
+	 * Check if uploads directory is public.
+	 *
+	 * @since 4.2.0
+	 * @return bool
+	 */
+	protected static function is_uploads_directory_is_public() {
+		$uploads = wp_upload_dir( null, true );
+
+		// Skip if returns an error.
+		if ( $uploads['error'] ) {
+			return true;
+		}
+
+		// Check for uploads root.
+		$baseurl_response = wp_safe_remote_get( $uploads['baseurl'] );
+		$baseurl_code     = intval( wp_remote_retrieve_response_code( $baseurl_response ) );
+
+		// Check for latest uploads sub directory.
+		// Double check in case the uploads root is only protected by an index.php or index.html file.
+		$url_response = wp_safe_remote_get( $uploads['url'] );
+		$url_code     = intval( wp_remote_retrieve_response_code( $url_response ) );
+
+		return 200 === $baseurl_code && 200 === $url_code;
```

Comment:
What status code do directory indexes return?

Reply:
200

Reply:
shouldn't there be an `or` instead of `and` in this condition? I would think that if any of those return 200, then something is publicly accessible?